test/FsAutoComplete.Tests.Lsp/Helpers.fs

```diff
@@ -539,11 +539,15 @@ let inline expectExitCodeZero (r: BufferedCommandResult) =
     0
     $"Expected exit code zero but was %i{r.ExitCode}.\nStdOut: %s{r.StandardOutput}\nStdErr: %s{r.StandardError}"
 
-let dotnetRestore dir =
-  runProcess dir "dotnet" "restore" |> Async.map expectExitCodeZero
-
-let dotnetToolRestore dir =
-  runProcess dir "dotnet" "tool restore" |> Async.map expectExitCodeZero
+let dotnetRestore dir = async {
+  let! r = runProcess (DirectoryInfo(dir).FullName) "dotnet" "restore -v d"
+  return expectExitCodeZero r
+}
```

Comment:
`Async.map` actually makes it hard to get correct stack tracing.

This was throwing on debug attaching for some reason. I was getting:

```
   Assembly loaded during TaskRun (NuGet.Build.Tasks.GetRestoreSettingsTask): System.Diagnostics.StackTrace, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a (location: C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\7.0.11\\System.Diagnostics.StackTrace.dll, MVID: a66c8790-3b61-402e-b635-09e18e1fbd38, AppDomain: [Default])
         System.ArgumentNullException: Value cannot be null. (Parameter 'path1')
            at System.ArgumentNullException.Throw(String paramName)
            at System.IO.Path.Combine(String path1, String path2)
            at NuGet.Common.NuGetEnvironment.GetFolderPath(NuGetFolderPath folder)
            at NuGet.Configuration.XPlatMachineWideSetting..ctor()
            at NuGet.Build.Tasks.GetRestoreSettingsTask.<>c.<.cctor>b__85_0()
            at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
            at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
            at System.Lazy`1.CreateValue()
            at NuGet.Build.Tasks.RestoreSettingsUtils.ReadSettings(String solutionDirectory, String restoreDirectory, String restoreConfigFile, Lazy`1 machineWideSettings, SettingsLoadingContext settingsLoadContext)
            at NuGet.Build.Tasks.GetRestoreSettingsTask.Execute()
       Done executing task \"GetRestoreSettingsTask\" -- FAILED.
```